src/components/application_manager/test/commands/mobile/register_app_interface_request_test.cc

```diff
@@ -227,6 +265,10 @@ TEST_F(RegisterAppInterfaceRequestTest, DISABLED_Run_MinimalData_SUCCESS) {
   EXPECT_CALL(
       app_mngr_,
       ManageMobileCommand(MobileResultCodeIs(mobile_apis::Result::SUCCESS), _));
+  EXPECT_CALL(app_mngr_,
+              ManageMobileCommand(CheckLockScreenIconUrlNotification(
```

Comment:
@VVeremjova, Yes, I'm sure. At first command send a response about successful app registration, and then send a notification with url for lock screen icon.